libraries/fc/include/fc/utility.hpp

```diff
@@ -18,6 +18,8 @@
 #define MAX_NUM_ARRAY_ELEMENTS (1024*1024)
 #define MAX_SIZE_OF_BYTE_ARRAYS (20*1024*1024)
 
+static_assert(MAX_SIZE_OF_BYTE_ARRAYS == 20*1024*1024, "Changing MAX_SIZE_OF_BYTE_ARRAYS breaks consensus. Make sure this is expected");
```

Comment:
This will need to be ported to the fc repo as well

Reply:
I don't think this belongs in `fc` when `fc` is a separate repo. Someone might be using fc for something completely different. I think we should move this static_assert into libraries/chain/include/eosio/chain/config.hpp